### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ajl388**
## 👨‍💻 Changes proposed

Added more tutorials and fixed typo.

## ✔️ Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,8 +97,9 @@ An Ethereum client is the software that allows Ethereum nodes to read the block
 ## DAO Communities
 
 - [LearnWeb3DAO](https://www.learnweb3.io/) Curious about Web3 as a developer? Wonder what are NFTs, DAOs, DeFi, and Smart Contracts? This is the place for you!.
+- [Cadena](https://cadena.dev/) - Build smart contracts, dapps, on multiple chains with a community and earn a Cadena NFT Certificate.
 - [Developer DAO](https://developerdao.notion.site/Getting-Started-with-Developer-DAO-2bddd332c51a4957b0b83f60f9fa4ebe) Build web3 with friends. New membership is officially open.
-- [Buildspace](https://buildspace.so/) Onboarding devs to web3. Start building cool stuff right now, earn NFTs, access secret opportunities in crypto.
+- [Buildspace](https://buildspace.com/) Onboarding devs to web3. Start building cool stuff right now, earn NFTs, access secret opportunities in crypto.
```
Comment (Pradumnasaraf) [inline README.md:102]:
```suggestion
- [Buildspace](https://buildspace.so/) Onboarding devs to web3. Start building cool stuff right now, earn NFTs, access secret opportunities in crypto.
```

Hey `https://buildspace.so` is correct one, please chnage back to it 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

**Review by ewainy — CHANGES_REQUESTED** [review]
Great addition for Cadena 😊
As @Pradumnasaraf has pointed out, please amend buildspace link to buildspace.so

**Comment by FrancescoXX** [comment]
For now, I don't see the community (like a discord or Slack channel)

Please revert the Buildspace link

**Review by ajl388 — COMMENTED** [review]
Updated links.
Discord community: https://discord.gg/UQayXxzazc

**Pull request merged**